nx/include/switch/services/nfc.h

```diff
@@ -102,103 +103,120 @@ typedef enum {
     NfcMifareCommand_Store     = 0xC2,
 } NfcMifareCommand;
 
+typedef enum {
+    NfpAmiiboFlag_Valid                 = BIT(0),   ///< Initialized in system settings.
+    NfpAmiiboFlag_ApplicationAreaExists = BIT(1),   ///< Application area exists.
+} NfpAmiiboFlag;
+
+typedef enum {
+    NfpBreakType_Flush  = 0,
+    NfpBreakType_Break1 = 1,
+    NfpBreakType_Break2 = 2,
+} NfpBreakType;
+
 typedef struct {
-    u8  uuid[10];
-    u8  uuid_length;
-    u8  reserved1[0x15];
-    u32 protocol;
-    u32 tag_type;
-    u8  reserved2[0x30];
+    u16 year;
+    u8 month;
+    u8 day;
+} NX_PACKED NfpDate;
```

Comment:
Remove NX_PACKED where it's not needed.

Reply:
Done, checked that sizes are now fine with static_assert tests